packages/server/src/v1/pulse/infrastructure/database/pulse.repo.ts

```diff
@@ -112,4 +112,15 @@ export class PulseRepo {
       .orderBy('last_active', 'desc')
     return res
   }
+
+  async getLatestProject(): Promise<string | undefined> {
+    const res = await this.knex<CodeClimbers.Pulse>(this.tableName)
+      .select('project')
+      .whereNotNull('project')
+      .whereNotIn('project', ['', '<<LAST_PROJECT>>'])
+      .orderBy('created_at', 'desc')
```

Comment:
Let's add a constraint here to limit finding results outside of like a 15 minute window. 

Also, knex has an option here for .first() that'll make it so we don't need to do the array access. 

i.e. 
res?.project || ''